app/src/main/java/io/gnosis/safe/ui/settings/owner/OwnerEnterNameFragment.kt

```diff
@@ -26,6 +27,7 @@ import pm.gnosis.utils.hexAsBigInteger
 import java.math.BigInteger
 import javax.inject.Inject
 
+@ExcludeClassFromJacocoGeneratedReport
```

Comment:
are we not exluding files based on name? Can we exclude all *Fragment files from report?